<?xml version="1.0" encoding="UTF-8"?>
<!--Copyright 2024 The MathWorks, Inc.-->

<rsccat xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.0" locale="ja_JP" product="parallel" xsi:noNamespaceSchemaLocation="../../resources/schema/msgcat.xsd">
  <message>
    <entry key="ConnectionError" context="error">Failed to connect to remote host ''{0}''.</entry>
    <entry key="CopyFileDestinationNotWritableError" context="error">コピー先 ''{1}'' が書き込み可能でないため、''{0}'' をコピーできませんでした。</entry>
    <entry key="CopyFileSourceNotReadableError" context="error">''{0}'' をコピーできませんでした。読み取り権限を確認してください。</entry>
    <entry key="CopyFolderOntoFileError" context="error">''{0}'' を ''{1}'' にコピーできません。フォルダーをファイルにコピーすることは、サポートされていません。</entry>
    <entry key="DeleteRemoteStorageError">Unable to delete remote storage location because there are jobs being mirrored.</entry>
    <entry key="FileNotFoundError" context="error">''{0}'' という名前のファイルが見つかりません。</entry>
    <entry key="IDAlreadyExistsError" context="error">ID {0,number,integer} のジョブは既にミラー中であるため、ミラーを開始できません。</entry>
    <entry key="IDDoesNotExistError" context="error">ID {0,number,integer} のジョブはミラー中ではないため、ミラーの停止はできません。</entry>
    <entry key="LocalFileAccessError">Error accessing local files:\n{0}</entry>
    <entry key="RemoteClusterAccessGeneralError">An internal error occurred with RemoteClusterAccess:\n{0}</entry>
    <entry key="RemoteClusterAccessUnknownError" context="error">RemoteClusterAccess で内部エラーが発生しました。</entry>
    <entry key="RemoteFileAccessError">Error accessing remote files:\n{0}</entry>
  </message>
</rsccat>
